core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -945,6 +945,12 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val SHUFFLE_READ_AHEAD =
+    ConfigBuilder("spark.shuffle.read.ahead.enabled")
```

Comment:
I just don't think this is worth a config? when would it be a bad idea?

Reply:
Yeah, It's better to remove this configuration. 
Thanks